src/components/sdk/MeetingControls/docs/VideoInputBackgroundBlurControl.stories.mdx

```diff
@@ -0,0 +1,60 @@
+import { Props } from '@storybook/addon-docs/blocks';
+import VideoInputBackgroundBlurControl from '../VideoInputBackgroundBlurControl';
+
+<Meta title="SDK Components/MeetingControls/VideoInputBackgroundBlurControl" />
+
+# VideoInputBackgroundBlurControl
+
+The `VideoInputBackgroundBlurControl` component renders a `ControlBarButton` with pop over menu options to select through multiple video input sources.
+These video input sources are provided by the `useVideoInputs` hook.
+The `VideoInputBackgroundBlurControl` component also has a checkbox that will enable or disable Background Blur on the chosen video input source.
```

Comment:
Is it going to be a checkbox or a button with a check icon? It may confuse user / builders hence asking.